File: nodejs/7 kyu/157.js
// https://www.codewars.com/kata/5983cba828b2f1fd55000114/train/javascript

// Passed

function oddOne(arr) {
  let result = -1;
  let currentIdx = 0;
  for (const num of arr) {
    if (Math.abs(num) % 2 === 1) {
      result = currentIdx;
      break;
    }

    currentIdx++;
  }

  return result;
}

const output = oddOne([4, -8, 98, -12, -7, 90, 100]);
console.log(output);

File: python/8 kyu/157.py
# https://www.codewars.com/kata/5808e2006b65bff35500008f/train/python 

# Passed

from string import ascii_lowercase

def position(letter):
    position = ascii_lowercase.index(letter) + 1
    result = f"Position of alphabet: {position}"
    return result

output = position("a")
print(output)